clusters/ztp-policies/common/kustomization.yaml

```diff
@@ -11,6 +11,8 @@ generators:
  # yamllint disable-line rule:line-length
  # This is common to all RAN DU deployments for vRAN Solution Enablement's Synchronization Workgroup
  - common-policies/common-ranGen-vsesync.yaml
+ # This is common to all VSE Loki/Vector clusters
+ - common-policies/common-vseloki.yaml
```

Comment:
This should be under resources as this doesn't need to be generated by the plugin

Reply:
You are right, fixed.